libraries/python/README.md

```diff
@@ -1,41 +1,87 @@
 Python library for Standard Webhooks
 
+# Standard Webhook Specification
+
+Visit [Standard Webhook Specification](https://github.com/standard-webhooks/standard-webhooks/blob/main/spec/standard-webhooks.md) to understand webhooks specification details.
+
+To debug or simulate webhooks messages check [webhooks tools](https://www.standardwebhooks.com/#tools).
+
+
 # Example
 
 Verifying a webhook payload:
 
 ```python
-from standardwebhooks.webhooks import Webhook
+from standardwebhooks import Webhook
+from app.config import WEBHOOK_SECRET_B64
```

Comment:
I think it's better to have it just read from the environment (`os.getenv` or something), this code sample is less clear than what was there before ("is app something I should install?")

Reply:
Changed to 

```python
import os

from standardwebhooks import Webhook


WEBHOOK_SECRET_B64 = os.getenv('WEBHOOK_SECRET_B64')

wh = Webhook(WEBHOOK_SECRET_B64)
```